pkgs/development/python-modules/isbnlib/default.nix

```
@@ -27,7 +27,7 @@ buildPythonPackage rec {
   meta = with lib; {
     description = "Extract, clean, transform, hyphenate and metadata for ISBNs";
     homepage = "https://github.com/xlcnd/isbnlib";
-    license = licenses.lgpl3;
+    license = licenses.lgpl3Only;
```

It looks like `LGPLv3+` to me ([COPYRIGHT.txt](https://github.com/xlcnd/isbnlib/blob/dev/COPYRIGHT.txt#L5-L8) and in the file headers, e.g., [here](https://github.com/xlcnd/isbnlib/blob/dev/isbnlib/_core.py#L5-L10)). To be `LGPLv3`, the license statement doesn't have to contain "..., or (at your option) any later version". 

```suggestion
    license = licenses.lgpl3Plus;
```

---

You are right, I searched for "newer", not "later" and therefore missed it.

---

Oh true, but this file says gpl3Only: https://github.com/xlcnd/isbnlib/blob/dev/LICENSE-LGPL-3.0-only.txt

---

I opened https://github.com/xlcnd/isbnlib/issues/77.

---

Correcting the license in https://github.com/NixOS/nixpkgs/pull/119302.